_episodes_rmd/01-continuos-data.Rmd

```diff
@@ -86,6 +86,8 @@ ggplot(data = interviews_plotting)
 > We would like to inspect the data for a potential relationship between the number of
 > people living in a household (`no_members`) and the number of items owned by that
```

Comment:
I think it might be better to "correct" the spelling in the exercise description, i.e., use `no_membrs` instead of `no_members` to avoid (some) confusion. A note on the spelling might be helpful in addition to that.